randovania/games/samus_returns/logic_database/Surface - West.txt

```diff
@@ -180,21 +180,18 @@ Extra - asset_id: collision_camera_017
       All of the following:
           Baby Metroid
           Any of the following:
-              Spiderspark (Intermediate) and Can Spiderspark
+              Spiderspark (Beginner) and Can Spiderspark
+              Spider Ball and Simple IBJ
               All of the following:
-                  Lay Power Bomb
                   Any of the following:
-                      Space Jump
+                      Lay Power Bomb
+                      Mid-Air Morph (Beginner) and Lay Bomb
                       Lightning Armor and Simple IBJ
```

Comment:
Remove this line

Reply:
May I ask why? if you dont have mid-air morph or pbs for the sj and ice methods, it's another valid way of breaking the block.

Reply:
The Simple IBJ path is already taken care of above, so this is just a duplicate.

Reply:
if you have spider it is. if you dont have spider, you need the sj or ice methods.

Reply:
I guess it reads weird to me since it says:
`Lightning Armor and Simple IBJ and Space Jump and Movement (Advanced)`
or
`Lightning Armor and Simple IBJ and Lightning Armor and Stand on Frozen Enemy (Intermediate) and Fully Freeze Enemy`

Reply:
Would it help if i added comments in each or clause? First one stating "Break the bomb block" and the second "Break the baby blocks"

Reply:
Comments would be good yes